src/app/public/modules/demo-page/demo-page-type-definitions.component.html

```diff
@@ -1,55 +1,57 @@
-<sky-docs-demo-page-section *ngIf="types.components.length"
-  heading="Components"
->
-  <sky-docs-directive-definition *ngFor="let config of types.components"
-    [config]="config"
-  ></sky-docs-directive-definition>
-</sky-docs-demo-page-section>
+<ng-template [ngIf]="types">
```

Comment:
This is the only functional change of the file, everything else is related to tab changes.